lainnya/abstrak-en.tex

```diff
@@ -0,0 +1,37 @@
+\begin{center}
+	\large
```

Comment:
accidentally using tab instead of spaces?

Reply:
seems still unresolved @messier12

Reply:
done @threeal